libs/remix-debug/test/decoder/vmCall.ts

```diff
@@ -39,31 +40,28 @@ export function sendTx (vm, from, to, value, data, cb) {
 }
 
 function createVm (hardfork) {
-  // const stateManager = new StateManagerCommonStorageDump({})
-  // stateManager.checkpoint(() => {})
-  const vm = new EthJSVM({
-    activatePrecompiles: true,
-    hardfork
-  })
-  vm.blockchain.validate = false
+  const common = new Common({ chain: 'mainnet', hardfork })
+  const vm = new VM({ common })   
```

Comment:
ideally you should use `await VM.create({ common })` here to await the init method for proper initialization, or you can use `await vm.init()`